common/lib/xmodule/xmodule/js/src/video/09_bumper.js

```diff
@@ -57,7 +59,8 @@ define('video/09_bumper.js',[], function () {
         },
 
         skip: function () {
-            this.element.trigger('skip');
+            this.element.trigger('skip', [this.doNotShowAgain]);
```

Comment:
> why doNotShowAgain?

Events plugin expects the flag: https://github.com/raccoongang/edx-platform/blob/anton/fix-events/common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js#L87